### DISCUSSION TITLE
Support templating for parameters argument of DatabricksSqlOperator

### DISCUSSION THREAD

**Discussion by khgouldy** [apache/airflow#32252] [Q&A]
### Description

The parameters arg of the parent class SQLExecuteQueryOperator is templated. I'm unsure of any difficulties making the parameters arg of DatabricksSqlOperator templated as well. However, this should be supported, and is what airflow sql operator users are used to.

### Use case/motivation

Use airflow template variables with params to easily pass {{ ds }} and other values to SQL queries on databricks.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by eladkal** [comment]
What parameter are you referring to?
I think https://github.com/apache/airflow/pull/32253 will fix your problem?

**Reply by potiuk** [reply]
And yes. If you need to add more fields there @khgould  - fel free to make PR adding those that are worth it. There is no need to open issue for those. just add PR. Adding templated fields is almost as easy as creating an issue, you could likely even do it without leaving the UI of GitHub by editing the right code in place - it will open a PR for you. You can see a number of similar smalll PRs in the past (look for "template_fields" in PRs).

**Comment by khgouldy** [comment]
Thanks for the quick response @jarek. You are the man!
PR <https://github.com/apache/airflow/pull/32263> submitted.
